source/extensions/extensions_metadata.yaml

```diff
@@ -3,21 +3,29 @@ envoy.access_loggers.file:
   - envoy.access_loggers
   security_posture: robust_to_untrusted_downstream
   status: stable
+  type_urls:
+  - envoy.extensions.access_loggers.file.v3.FileAccessLog
 envoy.access_loggers.extension_filters.cel:
   categories:
-  - envoy.access_loggers.extension_filters
+  - envoy.access_logger.extension_filters
```

Comment:
The removal is because of the mis-match in the code and in the docs. I aligned to the code variant because that seems harder to change. I agree the name is not perfect, which is why there was a typo probably.

Reply:
The category name is defined here https://github.com/envoyproxy/envoy/blob/0fae6970ddaf93f024908ba304bbd2b34e997a51/source/common/access_log/access_log_impl.h#L272.

Reply:
i wonder if there is a way we can catch issues like this

Reply:
This PR is pretty good at catching mis-labelled extensions and categories :) The test would fail without the fix.

Reply:
> The removal is because of the mis-match in the code and in the docs. I aligned to the code variant because that seems harder to change. I agree the name is not perfect, which is why there was a typo probably.

I'm probably missing something here, why will it be harder to change the code?
Is it because the category will be sent to the servers (as part of the bootstrap's [extensions](https://github.com/envoyproxy/envoy/blob/b49eb0f7ba6061d70b41b8b5b469e0c4cba28ebc/api/envoy/config/core/v3/base.proto#L119)) and the server won't know about the extension?

Reply:
@adisuissa Yes, and I haven't seen the usage of deprecated names for categories. Changing extension name is certainly risky since they can still be referred by name from xDS. Category names are less risky, so if you agree, we can change it the other way, fix in code to `envoy.access_loggers.extension_filters`.

Reply:
This is a very new category, so I'm going to take the liberty to rename the code version.